src/assets/images/skewed-triangle.svg

```diff
@@ -0,0 +1,13 @@
+<svg
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   height="101.17517"
+   width="92.936852"
+   version="1.1"
+   viewBox="0 0 92.936852 101.17517">
+  <polygon
+     transform="rotate(-156,39.752044,60.969868)"
+     points="100,100 0,100 50,13.397 "
+     id="tri"
+     style="fill:none;stroke:#0060DF;stroke-opacity:1;stroke-width:0.5" />
```

Comment:
might wanna set the stroke to "currentColor"?

Reply:
`currentColor` does not work in an non-embedded svg (see the other svgs in that folder).